pkg/sfu/streamallocator.go

```diff
@@ -459,12 +459,14 @@ func (s *StreamAllocator) handleSignalEstimate(event *Event) {
 
 	s.prevReceivedEstimate = s.receivedEstimate
 	s.receivedEstimate = int64(remb.Bitrate)
-	if s.prevReceivedEstimate != s.receivedEstimate {
-		s.logger.Debugw("received new estimate",
-			"old(bps)", s.prevReceivedEstimate,
-			"new(bps)", s.receivedEstimate,
-		)
-	}
+	/*
```

Comment:
Reducing log spam as @frostbyte73 had expressed concern about this being too noisy in a different PR.